rasterio/crs.pyx

```diff
@@ -914,38 +922,23 @@ cdef class CRS:
         return self.to_string()
 
     def __repr__(self):
-        epsg_code = self.to_epsg()
-        if epsg_code:
-            return "CRS.from_epsg({})".format(epsg_code)
+        if self._epsg:
+            return "CRS.from_epsg({})".format(self._epsg)
         else:
             return "CRS.from_wkt('{}')".format(self.wkt)
 
     def __eq__(self, other):
-        cdef OGRSpatialReferenceH osr_s = NULL
-        cdef OGRSpatialReferenceH osr_o = NULL
         cdef CRS crs_o
+        cdef const char* options[2]
+        options[0] = b"IGNORE_DATA_AXIS_TO_SRS_AXIS_MAPPING=YES"
```

Comment:
Curious about `YES` 
in https://github.com/rasterio/rasterio/pull/3208 we will set default to `NO` 

 https://github.com/rasterio/rasterio/pull/3208/files#diff-a25cc236cc45436e2c537cdec6d43f99459569dab794dd16a642656b53b5c4d9R955

I though we wanted to keep
```
CRS.from_epsg(4326) == CRS.from_string("OGC:CRS84")
>>> False
```

Maybe I'm mistaking but if we `ignore axis order` ☝️  will return `True` 

I also don't see any tests for this equality, I feel we should add one in this PR!

Reply:
yeah the tests I added to #3208 should be included here, although yeah that doesn't sound like the decision that was made in the other PR and is more like my initial implementation here